integration-libs/opf/occ/adapters/occ-opf.adapter.ts

```diff
@@ -31,10 +39,10 @@ export class OccOpfAdapter implements OpfAdapter {
   ) {}
 
   getActiveConfigurations(): Observable<ActiveConfiguration[]> {
-    const headers = new HttpHeaders({
-      'sap-commerce-cloud-public-key':
-        this.config.opf?.commerceCloudPublicKey || '',
-    });
+    const headers = new HttpHeaders().set(
```

Comment:
Maybe it is better to create interceptor to handle this headers? i see the code repetition in initiatePayment method